Formula/caf.rb

```diff
@@ -27,11 +27,11 @@ class Caf < Formula
   fails_with gcc: "5"
 
   def install
-    mkdir "build" do
-      system "cmake", "..", *std_cmake_args, "-DCAF_ENABLE_TESTING=OFF"
-      system "make"
-      system "make", "install"
-    end
+    system "cmake", "-S", ".", "-B", "build", *std_cmake_args,
+                    "-DCMAKE_INSTALL_RPATH=#{lib}",
+                    "-DCAF_ENABLE_TESTING=OFF"
```

Comment:
```suggestion
    tools = pkgshare/"tools"
    args = ["-DCAF_ENABLE_TESTING=OFF"]
    args << "-DCMAKE_INSTALL_RPATH=#{rpath};@loader_path/#{lib.relative_path_from(tools)}" if OS.mac?
    system "cmake", "-S", ".", "-B", "build", *std_cmake_args, *args
```
Using a relative path is better because we save the overhead of `RPATH` rewriting at bottle/pour time.